packages/handlersjs-core/lib/storage/memory-store.spec.ts

```diff
@@ -18,111 +18,123 @@ describe('MemoryStore', () => {
 
   });
 
-  it('should not iterate over keys pairs when it is empty', async () => {
+  describe('constructor', () => {
```

Comment:
Since we put every other method in a separate describe block, you can just leave the constructor test(s) in the top-level one.